python/setup.py

```diff
@@ -100,12 +96,6 @@
     if len(sys.argv) == 1:
         sys.exit(0)
 
-##############################################################################
-# - Cloning RAFT and dependencies if needed ----------------------------------
-
-# Use RAFT repository in cuml.raft
-
-raft_include_dir = use_raft_package(raft_path, libcuml_path)
```

Comment:
We should be able to delete the code of this function in setuputils.py as well

Reply:
I kept this around only as a potential for pinning the Python package in the future. I suppose we could remove this and just use the conda package labels for raft like we do w/ libcumlprims.